session/session_test.go

```diff
@@ -3,21 +3,25 @@ package session
 import (
 	"testing"
 	"time"
-
+	zookeeper "github.com/Shopify/gozk"
 	toxiproxy "github.com/Shopify/toxiproxy/client"
 )
 
-func TestReceiveEventWhenSubscribing(t *testing.T) {
+func Setup(t *testing.T) (*toxiproxy.Proxy){
```

Comment:
I'd call this `createProxy` instead of `Setup`, since all it's really doing is creating a new proxy.